lte/gateway/python/magma/enodebd/BUILD.bazel

```diff
@@ -32,10 +32,10 @@ py_binary(
     visibility = ["//visibility:public"],
     deps = [
         ":enodebd_lib",
+        "//lte/gateway/python/magma/enodebd/tr069:server",
         "//lte/protos:mconfigs_python_proto",
         "//orc8r/gateway/python/magma/common:sentry",
         "//orc8r/gateway/python/magma/common:service",
-        "//lte/gateway/python/magma/enodebd/tr069:server",
```

Comment:
Thanks for fixing the ordering on these! Don't know how they slipped in, buildifier should have caught them on prior PRs.